### DISCUSSION TITLE
Removing `level_id` from Users' Model

### DISCUSSION THREAD

**Discussion by cjmellor** [cjmellor/level-up#65] [Announcements]
I've realised that having the `level_id` in the `users` table is absolutely pointless. The level data is stored in the `experiences` table, and the `getLevel()` method is using that, so that `level_id` data in the `users` table is just extra fluff that isn't needed.

I aren't sure if this would be classed as a breaking change 🤔

**Comment by Supernova3339** [comment]
I agree, if it doesn't affect anything, is it safe to remove the migration? If so, which one? Cheers 🎉 

EDIT: Remove the one that connects the relation to the users table ( `add_level_relationship_to_users_table` )
